### DISCUSSION TITLE
How can I add more publications while I are already started screening in ASReview?

### DISCUSSION THREAD

**Discussion by Tsklopper** [asreview/asreview#562] [Q&A] [answered]
During the screening process, I came across some additional data which could be relevant for my research. How can I add more publications while I are already started screening in ASReview?

**Comment by terrymyc** [comment, marked as answer]
You can export the results of ASReview you have so far and add your new publications to this file. Make sure that for your new publications the column included is empty and rename or delete the column of `record_id`. This latter will save you from running into errors. Then simply import this updated file to ASReview and you can continue the screening process.
